src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -232,6 +232,28 @@ def state_transition(docs):
     out += '</ul></fieldset>'
     return out
 
+def priority_transition(docs):
+    "create html for priority transition format"
+    if  not isinstance(docs, list):
```

Comment:
extra space here. Maybe just run the "reformat code" in this module as another commit.

Reply:
thanks I will fix that.